### PR TITLE
Collected small changes

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This pull request combines multiple small fixes and updates

**Related Issue(s)**

N/A

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

No known issues.

**Implementation Notes**

The following individual changes are included:
- silence compiler warnings in the GPU package library
- add check to various qeq fixes so that they print a warning if applied to a non-charge neutral set of atoms. document the warning
- add missing initializers for multiple fixes to address coverity scan warnings
- use symbolic constants instead of "magic" numbers in fix orient/fcc and fix orient/bcc. address out-of-bound access issue.
- improve formatting and readability of SHAKE statistics by adding Bond/Angle prefix and using a dynamic column width for the first column

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system

### CODE REVIEW

**Comment by akohlmey** [comment]
@athomps this PR includes changes to check for non-neutral fix groups on any of the QEq fixes and prints a warning (threshold is the same as for KSpace) and also copies the note from the fix qeq doc page to the others.
Do you think it should be a sentence added to explain that the fixes will enforce neutrality during self-consistency iterations?

**Comment by athomps** [comment]
Sorry, I think I changed it slightly after you updated, but your update is good. No need to update again.

**Comment by akohlmey** [comment]
> I suggest instead:

thanks. I updated the doc files accordingly. To avoid ambiguity, I mentioned that this applies to the fix group. In most cases the fix group will be "all", but just in case it isn't.

File: doc/src/Build_windows.rst [hunk 1]
```diff
@@ -77,12 +81,13 @@ Following the integration of the :doc:`platform namespace
...
-CMake support of Visual Studio or use an external CMake installation to
-create build files for the Visual Studio build system.  Please note that
+16) and Visual Studio 2022 (aka version 17).  Not all features and
+packages in LAMMPS are currently supported out of the box, but a preset
+``cmake/presets/windows.cmake`` is provided that contains the packages
+that have been compiled successfully.  You **must** use the CMake based
+build procedure, and either use the integrated CMake support of the
+Visual Studio IDE or use an external CMake installation to create build
+files for the Visual Studio build system.  Please note that
 on launching Visual Studio it will scan the directory tree and likely
```
[the remark this answers is not in the document]
Comment (rbberger) [reply doc/src/Build_windows.rst:91]: @akohlmey yeah, since CMake support is so fresh I would recommend the preview builds at the moment.

**Comment by akohlmey** [comment]
@sjplimp @athomps @stanmoore1 FYI this PR changes the statistics output for fix shake and fix rattle from
```
SHAKE stats (type/ave/delta/count) on step 300
     4   1.111     8.82687e-06       18
     6   0.997003  1.07911e-05        6
     8   1.08      2.38249e-06        9
    10   1.111     7.15472e-06       12
    12   1.08      1.0566e-05         9
    14   0.960007  0                  1
    18   0.957203  1.26657e-05     1914
    31   104.52    0.00191259       638
```
to
```
SHAKE stats (type/ave/delta/count) on step 300
Bond:    4   1.111     8.82687e-06       18
Bond:    6   0.997003  1.07911e-05        6
Bond:    8   1.08      2.38249e-06        9
Bond:   10   1.111     7.15472e-06       12
Bond:   12   1.08      1.0566e-05         9
Bond:   14   0.960007  0                  1
Bond:   18   0.957203  1.26657e-05     1914
Angle:  31   104.52    0.00191259       638

```

**Review by sjplimp — APPROVED** [review]

**Comment by sjplimp** [comment]
New format looks nice!

**Pull request merged**